web/ui/assets_nobuiltin.go

```diff
@@ -0,0 +1,11 @@
+//go:build !builtinassets
+// +build !builtinassets
+
+package ui
+
+import "net/http"
+
+// Assets contains the UI's assets.
+func Assets() http.FileSystem {
+	return http.Dir("./web/ui/build")
```

Comment:
Does this work on windows?

Reply:
According to the docs this should be split by filepath.Separator, thanks.